build_tarballs.jl

```diff
@@ -49,7 +49,7 @@ elif [[ "${target}" == x86_64-w64-mingw32 ]]; then
 elif [[ "${target}" == x86_64-unknown-* ]]; then
     export CROSS=x86_64-unknown-gcc
     export CC=/opt/${target}/bin/${target}-gcc
-    export CXX=/opt/${target}/bin/${target}-g++
+    export CXX=/opt/${target}/bin/${target}-clang++
```

Comment:
`CC` and `CXX` should already be set appropriately; have you tried just not setting them?

Reply:
If you have, what breaks?  In order to use `clang` you need to provide `-target` and `--sysroot` arguments; here's what the defaults are for targeting FreeBSD:

```
# echo $CC
ccache /opt/x86_64-linux-gnu/tools/clang -target x86_64-unknown-freebsd11.1 --sysroot /opt/x86_64-unknown-freebsd11.1/x86_64-unknown-freebsd11.1/sys-root
```